ctru-rs/src/lib.rs

```diff
@@ -25,19 +53,24 @@ macro_rules! from_impl {
     };
 }
 
-/// Activate the default panic handler.
+/// Activate the custom [`ctru-rs`](crate) panic handler.
 ///
-/// With this implementation, the main thread will stop and try to print debug info to an available [console::Console].
-/// In case it fails to find an active [console::Console] the program will just exit.
+/// With this implementation, the main thread will stop and try to print debug info to an available [`Console`](console::Console).
+/// In case it fails to find an active [`Console`](console::Console) the program will just exit.
 ///
 /// # Notes
 ///
-/// When ´test´ is enabled, this function won't do anything, as it should be overridden by the ´test´ environment.
+/// When `test` is enabled, this function will not do anything, as its behaviour should be overridden by the `test` environment.
 pub fn use_panic_handler() {
     #[cfg(not(test))]
     panic_hook_setup();
 }
 
+/// Internal protocol to activate the custom panic handler hook.
+///
+/// # Notes
+///
+/// When `test` is enabled, this function will be ignored.
```

Comment:
same here, don't think this needs to be a doc comment

Reply:
Doc comments work differently when building internal documentation (the docs that aren't publicly visible). As such, even though private members without docs don't trigger the `#[warning(missing_docs)]`, there is a way to build and read them (`cargo 3ds doc --document-private-items`).

Normal comments wouldn't be able to do that.